rpc.go

```diff
@@ -141,27 +141,34 @@ func IsNotFound(err error) bool {
 	return checkError(err, ErrNoDomain)
 }
 
+// isTemporary returns true if the error returned from a read is transiet.
+// If the error type is an OpError, check whether the net connection
+// error condition is temporary (which means we can keep using the
+// connection).
+// Errors not of the net.OpError type tend to be things like io.EOF,
+// syscall.EINVAL, or io.ErrClosedPipe (i.e. all things that
+// indicate the connection in use is no longer valid.)
+func isTemporary(err error) bool {
+	opErr, ok := err.(*net.OpError)
+	if ok {
```

Comment:
feel free to ignore; i think we can simplify this to

```go
if ok {
        return opErr.Temporary()
}

return false
```

Reply:
eh, it's fine, I have to fix the typo anyway